src/components/input/demo/example.twig

```diff
@@ -3,5 +3,5 @@
   {% block element %}
     {% include '../input.twig' %}
   {% endblock %}
-  {% include '../../button/button.twig' with { label: 'Button' } %}
+  {% include '../../button/button.twig' with { label: 'Button' } only %}
```

Comment:
@spaceninja For my own learning purposes, what does `only` do? I don't know. 🙂

Reply:
We pass in parameters to the example file, and by default it passes those parameters down. The net effect is that I was setting `class="js-elastic-textarea"` on the Story, and it was applying the class to both the textarea and the button.

Adding `only` here says "only pass in the parameters I've specified" — in this case, only pass in the `label` parameter.

Tyler told me about this when I mentioned the problem. Details here: https://twig.symfony.com/doc/2.x/tags/include.html